Jetchat/buildSrc/src/main/java/com/example/compose/jetchat/buildsrc/dependencies.kt

```diff
@@ -57,7 +57,7 @@ object Libs {
 
         object Compose {
```

Comment:
Please update Material 3 to 1.0.0-alpha06

Reply:
Nope, I meant https://maven.google.com/web/index.html#androidx.compose.material3:material3:1.0.0-alpha06. For some reason I couldn't comment inline on line 78

Reply:
My mistake, thanks for clarifying! I updated it to alpha06. The one change affecting jetchat that I saw was NavigationDrawer was renamed to ModalNavigationDrawer, so I went ahead and updated that too.